spectrocloud/resource_cluster_aws.go

```diff
@@ -27,30 +28,30 @@ func resourceClusterAws() *schema.Resource {
 
 		SchemaVersion: 2,
 		Schema: map[string]*schema.Schema{
-			"name": {
+			Name: {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-			"cluster_profile_id": {
+			ClusterProfileId: {
```

Comment:
@saamalik, At few places, I have changed string value to a defined constant. Few places are still left to be changed.